xml/ha_hawk2_manage_i.xml

```diff
@@ -0,0 +1,365 @@
+     </para>
+    </step>
+    <step>
+     <para>Stop the resource:</para>
+     <substeps>
+      <step>
+       <para> In the left navigation bar, select <guimenu>Status</guimenu>. The list of
+         <guimenu>Resources</guimenu> also shows the <guimenu>Status</guimenu>. </para>
+      </step>
+      <step>
+       <para>In the <guimenu>Operations</guimenu> column click the <guimenu>Stop</guimenu> button
+        next to the resource. </para>
+      </step>
+      <step>
+       <para>To continue, confirm the message that appears.</para>
+       <para>The <guimenu>Status</guimenu> column will reflect the change as soon as the resource is
+        stopped.</para>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>Delete the resource:</para>
+     <substeps>
+      <step>
+       <para> In the left navigation bar, select <guimenu>Edit</guimenu>. </para>
+      </step>
+      <step>
+       <para>In the list of <guimenu>Resources</guimenu>, go to the respective resource. From the
+         <guimenu>Operations</guimenu> column click the <guimenu>Delete</guimenu> icon next to the
+        resource.</para>
+      </step>
+      <step>
+       <para>To continue, confirm the message that appears.</para>
+      </step>
+     </substeps>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.conf.hawk2.manage.migrate">
+   <title>Migrating Cluster Resources</title>
+   <para> As mentioned in <xref linkend="sec.conf.hawk2.failover"/>, the cluster will fail over
+    (migrate) resources automatically in case of software or hardware failures&mdash;according
+    to certain parameters you can define (for example, migration threshold or resource stickiness).
+    Apart from that, you can manually migrate a resource to another node in the cluster. Or you
+    decide to move it away from the current node and leave the decision about where to put it to the
+    cluster. </para>
+   <procedure xml:id="pro.hawk2.rsc.migrate">
+    <title>Manually Migrating a Resource</title>
+    <step>
+     <para>Log in to &hawk;2:</para>
+     <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
+    </step>
+    <step>
+     <para> In the left navigation bar, select <guimenu>Status</guimenu>. The list of
+       <guimenu>Resources</guimenu> also shows the <guimenu>Status</guimenu>. </para>
+    </step>
+    <step>
+     <para>In the list of <guimenu>Resources</guimenu>, go to the respective resource.</para>
+    </step>
+    <step>
+     <para><remark>taroth 2015-12-07: krig, is there any way to select which node to migrate the
+       resource to? krig: also, Hawk1 has two options: "Move" (lets you select where to move the rsc
+       to and creates a location constraint with an INFINITY score for the destination node) and
+       "Away from current node" (creates a location constraint with a -INFINITY score for the
+       current node) - si there anything like this with Hawk2? </remark>In the
```

Comment:
Done, filed https://bugzilla.suse.com/show_bug.cgi?id=958386 (this also covers your note right above this one).